Makefile

```diff
@@ -90,6 +90,14 @@ generate-in-docker:
 format: $(GOIMPORTS) $(GOIMPORTSREVISER)
 	@bash $(GARDENER_HACK_DIR)/format.sh ./cmd ./pkg ./test
 
+.PHONY: sast
```

Comment:
Similar to the already deleted comment from @plkokanov. If the gardener repo is cloned under repo-root/gardener, then `make sast` is failing. We need also the exclude dirs to be configurable in the original script.